devstack_vm/bin/gerrit-git-prep.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash -e
```

Comment:
please update it based on the latest upstream (https://github.com/openstack-infra/project-config/blob/master/jenkins/scripts/gerrit-git-prep.sh)